packages/template-retail-react-app/app/pages/product-detail/index.jsx

```diff
@@ -99,8 +99,7 @@ const ProductDetail = () => {
                     'prices',
                     'variations',
                     'set_products',
-                    'bundled_products',
-                    'page_meta_tags'
```

Comment:
Should page_meta_tags be included, no? 🤔

Reply:
Ahh yes. We took that out from 3.9.0 but the corresponding SCAPI changes required for page meta tags will go live in 2 days so we can re-add it then.